### PR TITLE
hotfix: Improved metrics

### PR DESCRIPTION
**Pull request by Shahar-Y**
Added appID and AuthType to the metrics, with an option to add mre info from claims.
No breaking changes, and tested with kibana locally
Signed-off-by: shahar-y <shya95@gmail.com>

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3'
+version: '2.1'
```
Comment (yoavrozner) [inline docker-compose.yml:1]: Why?

**Review by yoavrozner — CHANGES_REQUESTED** [review]
Dear @Shahar-Y, please remove the dropbox changes because it is impossible to review both changes that are unrelated to each other. Furthermore, If I would approve it, the dropbox changes might not get reviewed and still be in the develop. Quick solution: open a new branch from dropbox branch, and open PR from `hotfix: Improved metrics` to the new branch and the changes should not appear.

**Comment by Shahar-Y** [comment]
> Dear @Shahar-Y, please remove the dropbox changes because it is impossible to review both changes that are unrelated to each other. Furthermore, If I would approve it, the dropbox changes might not get reviewed and still be in the develop. Quick solution: open a new branch from dropbox branch, and open PR from `hotfix: Improved metrics` to the new branch and the changes should not appear.

But that's what I did, here: https://github.com/meateam/api-gateway/pull/220

**Comment by Shahar-Y** [comment]
Closing in favor of #223

**Pull request closed without merging**